recipes/fast-double-parser/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
```

Comment:
```suggestion
        if not tools.cross_building(self):
```

self.settings is outdated, use only self, this new signature is better better compatible withe build context (dual profile)

Reply:
Thank you! I fixed it.